website/client/app.vue

```diff
@@ -13,34 +13,44 @@ div
     snackbars
     router-view(v-if="!isUserLoggedIn || isStaticPage")
     template(v-else)
-      template(v-if="isUserLoaded")
-        notifications-display
-        app-menu
-        .container-fluid
-          app-header
-          buyModal(
-            :item="selectedItemToBuy || {}",
-            :withPin="true",
-            @change="resetItemToBuy($event)",
-            @buyPressed="customPurchase($event)",
-            :genericPurchase="genericPurchase(selectedItemToBuy)",
-
-          )
-          selectMembersModal(
-            :item="selectedSpellToBuy || {}",
-            :group="user.party",
-            @memberSelected="memberSelected($event)",
-          )
-
-          div(:class='{sticky: user.preferences.stickyHeader}')
-            router-view
+        template(v-if="isUserLoaded")
```

Comment:
can you fix the indentation here? Then this will be ready

Reply:
I mean, there seem to be an extra indent used here

Reply:
oh you are right! fixing..